README.md

```diff
@@ -18,3 +18,7 @@ pgAdmin interface is served at localhost:5050 with username admin@admin.com and
 2. `source ./venv/bin/activate`
 3. `pip install -r requirements.txt`
 4. `pip install pre-commit pylint && pre-commit install`
+
+## To load departments and privileges from fixtures
+`python manage.py loaddata api/fixtures/Department.json --app api.Department`
+`python manage.py loaddata api/fixtures/ProjectMemberPrivilege.json --app api.ProjectMemberPrivilege`
```

Comment:
Change the command to load everything from fixtures

Reply:
done